pbeApp/pbexport.cpp

```diff
@@ -285,6 +306,58 @@ void transcode_samples(PBWriter& self)
     std::cerr<<"Wrote "<<nwrote<<"\n";
 }
 
+void PBWriter::skipForward(const char* file)
+{
+	//find the last sample that was written into the given file and skip forward the reader to the first
+	//sample that has a timestamp later than the last sample in the file
+	std::ifstream inpstr(file);
+	std::string temp;
+	char *bfr;
+	const char *str;
+	//Doesn't matter which data type we choose, because we are interested only into timestamp,
+	//which is stored at the beginning of the event, so it's the same for all types.
+	//Sometimes protobuf prints a warning, because a value is missing, but the timestamp is still there
+	EPICS::VectorString en;
```

Comment:
This is a rather questionable statement as it depends on the (almost certainly unspecified) behavior of the PB library when decoding fails.  Since the file is already being read from start it seems straightforward decode the PayloadInfo to get the correct type.